sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/StateStoreSuite.scala

```diff
@@ -64,21 +64,122 @@ class StateStoreSuite extends StateStoreSuiteBase[HDFSBackedStateStoreProvider]
     require(!StateStore.isMaintenanceRunning)
   }
 
+  def updateVersionTo(provider: StateStoreProvider, currentVersion: => Int,
+                      targetVersion: Int): Int = {
```

Comment:
Thanks for correcting style guide. Will fix.
Regarding `currentVersion: => Int` is somehow I was trying to reuse currentVersion and didn't roll back. Will fix.
And I agree it would be better to have `incrementVersion` to shorter the code. Will address.